src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -369,18 +370,25 @@
 
 (defn- community-fetching-placeholder
   [id]
-  (let [fetching? (rf/sub [:communities/fetching-community id])]
+  (let [theme     (quo.theme/use-theme)
+        top-inset (safe-area/get-top)
+        fetching? (rf/sub [:communities/fetching-community id])]
     [rn/view
-     {:style               style/fetching-placeholder
+     {:style               (style/fetching-placeholder top-inset)
       :accessibility-label (if fetching?
                              :fetching-community-overview
                              :failed-to-fetch-community-overview)}
-     [not-implemented/not-implemented
-      [rn/text
-       {:style style/fetching-text}
-       (if fetching?
-         "Fetching community..."
-         "Failed to fetch community")]]]))
+     [quo/page-nav
+      {:title      "Community Overview"
+       :type       :title
+       :text-align :left
+       :icon-name  :i/close
+       :on-press   #(rf/dispatch [:navigate-back])}]
+     [quo/empty-state
+      {:image           (resources/get-themed-image :cat-in-box theme)
+       :description     (when-not fetching? (i18n/label :t/here-is-a-cat-in-a-box-instead))
+       :title           (if fetching? "Fetching community..." "Failed to fetch community")
```

Comment:
@Parveshdhull, shouldn't this be in our translation files?

Reply:
Ah yeah good catch 🙏
Most likely should be in the translation file

Reply:
hi @ajayesivan, @seanstrom, Thank you for reviewing PR.

I didn't move them intentionally, as they were already in the PR and I assumed that's what we do when we don't have proper designs and implementation is just temporary. wdyt, should we move them to the translations file or handle them when implementing this placeholder when we have proper designs?

Reply:
I see what you mean about these labels being temporary 👍

My only thought is that if this UI can be seen without a feature flag, then maybe the labels should be translatable. And later on if we want to change them we can decide to remove them or not. How does that sound?

Reply:
sounds good, will move them to translations file in one of my PRs. thank you

Reply:
Thank you @Parveshdhull 🙏